### ISSUE TITLE
A11y_.NETOnlineREPL_.NETEditor_NonTextContrast: Color contrast ratio of the focus indication over the code suggestion list item is less than 3:1.

### ISSUE THREAD

**Issue by pranshu512** [dotnet/try#912]
“[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yWCAG2.1;#A11yTCS;#A11ySev2;#BM_.NET Online REPL_Web_Feb2021;#.NET Online REPL;#Win10;#FTP;#DesktopWeb;#WCAG1.4.11;#Chrome;

### **Environment Details:**
Application Name: .NET Online REPL
Operating System: Windows10
Browser Version : Google Chrome Version 87.0.4280.141 (Official Build) (64-bit)

### **Repro Steps:**

1. Open .NET Editor coding screen .
2. Tab to .NET Editor section and type in NET Editor. 
3. Check whether color contrast ratio of the focus indication over the code suggestion list item is less than 3:1 or not.

### **Actual:**
Color contrast ratio of the focus indication over the code suggestion list item which appear after typing any letter is less than 3:1.

### **Expected:**
Color contrast ratio of the focus indication over the code suggestion list item should be more than 3:1.

### **User Impact:**
Low vision user will not be able to see the focus on code suggestion list item.

**Comment by fsteffi** [comment]
still reproduces:
https://learn.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/tutorials/hello-world?tutorial-step=1&WT.mc_id=-blog-scottha

**Comment by AbhitejJohn** [comment]
@fsteffi : Logged https://github.com/MicrosoftDocs/feedback/issues/3901 to track this on the docs team. There isn't much we can do here in the interim.

**Comment by fsteffi** [comment]
**GithubTags**:#External;External:MicrosoftDocs;#LinkedBug;#Linked:Bug3901;

external bughttps://github.com/MicrosoftDocs/feedback/issues/3901

@AbhitejJohn  as this bug is external to Microsoft Docs this bug shall be in open state until its external is fixed as per the external resolved bug process

**Comment by AnKushSingh05** [comment]
Verified the issue in below environment. Issue is fixed
https://learn.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/tutorials/hello-world?tutorial-step=1&WT.mc_id=-blog-scottha
![Screenshot (2289)](https://github.com/user-attachments/assets/4783aa60-78ae-4940-aaf3-b5be252340d4)
